src/frontend/src/utils/iiConnection.ts

```diff
@@ -393,10 +396,14 @@ export class Connection {
     });
   };
 
-  lookupRecovery = async (userNumber: UserNumber): Promise<DeviceData[]> => {
+  lookupRecovery = async (
```

Comment:
would be nice to explain why you're removing the `alias`, even though it's temporary